docs/docs/deployment/hosting/locally-api.md

```diff
@@ -360,6 +360,12 @@ services:
 If not running our application via docker, you can find gunicorn's documentation on statsd instrumentation
 [here](https://docs.gunicorn.org/en/stable/instrumentation.html)
 
+### GitHub Integration Environment Variables
+
+- `GITHUB_PEM`: Create a PEM in your GitHub App and paste it here.
+- `GITHUB_APP_ID`: Your GitHub App ID
+- `GITHUB_WEBHOOK_SECRET`: Your GitHub webhook secret
```

Comment:
We probably need to provide more information about how to retrieve these.

Reply:
I added additional information.